packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/variations.blade.php

```diff
@@ -157,19 +157,17 @@ class="control"
             </td>
 
             <td>
-                <div class="control-group" :class="[errors.has(variantInputName + '[images]') ? 'has-error' : '']">
+                <div class="control-group" :class="[errors.has(variantInputName + '[images][' + index + ']') ? 'has-error' : '']">
                     <div v-for='(image, index) in items' class="image-wrapper">
-                        <label class="image-item" v-bind:class="{ 'has-image': imageData[index] }">
+                        <label class="image-item" v-bind:class="{ 'has-image': imageData[index] }" style="height: 100px; width: 100px; background-size: 100px 100px;">
```

Comment:
@rahulshukla-webkul is there a specific reason why css code is added here instead of using the scss files?